Index: src/sentEmail.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/sentEmail.js b/src/sentEmail.js
new file mode 100644
--- /dev/null	(date 1777485542990)
+++ b/src/sentEmail.js	(date 1777485542990)
@@ -0,0 +1,10 @@
+export function openEmail() {
+    const email = "Katharina.Zwick@student.hpi.uni-potsdam.de";
+    const subject = encodeURIComponent("Frage zum Rettungsläufer Netzwerk");
+    const body = encodeURIComponent(
+        "Hallo,\n\nHallo, ich habe folgende Frage:\n\n"
+    );
+
+    window.location.href = `mailto:${email}?subject=${subject}&body=${body}`;
+}
+
